docs/big-graph-from-cypher.adoc

```diff
@@ -0,0 +1,248 @@
+sys     0m0.088s
+----
+
+=== Different output formats
+
+We try the different output formats, changing the `config` parameter `format`.
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword  
+"call apoc.export.cypher.all('yourPath/exportData.cypher', {format:'neo4j-shell'})"
+----
+
+==== Results
+
+.neo4j-shell
+
+----
+real    1m49.268s
+user    0m0.904s
+sys     0m0.072s
+----
+
+.cypher-shell
+
+----
+real    1m55.089s
+user    0m0.892s
+sys     0m0.092s
+----
+
+.plain
+
+----
+real    1m54.490s
+user    0m0.932s
+sys     0m0.076s
+----
+
+=== Many files (separateFiles config)
+
+With the param `separateFiles` (default false) we can export our graph or part of it, in different files.
+In the example below we name the exported file `exportAll.cypher` so our export will be:
+
+ * exportAll.cleanup.cypher
+ * exportAll.nodes.cypher
+ * exportAll.relationships.cypher
+ * exportAll.schema.cypher
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword  
+"call apoc.export.cypher.all('yourPath/exportAll.cypher', {format:'neo4j-shell',separateFiles:true})"
+----
+
+==== Result
+
+----
+real 1m55.229s
+user 0m0.960s
+sys  0m0.084s
+----
+
+
+== Export from query
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword
+"call apoc.export.cypher.query('MATCH (n) OPTIONAL MATCH p = (n)-[r]-(m) RETURN n,r,m',
+'yourPath/exportQuery.cypher', {format:'neo4j-shell', batchSize: 10000})"
+----
+
+=== Result
+
+----
+real    3m34.924s
+user    0m0.992s
+sys     0m0.068s
+----
+
+== Export from given nodes and rels
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword  "Match (n:Person)-[r:LIKES_COMMENT]->(c:Comment) CALL apoc.expor
+t.cypher.data([n],[r], 'yourPath/exportData.cypher',{format:'plain', batchSize:1000}) yield nodes, relationships return 'none'" > youPath/exportDataResult.cypher
+----
+
+=== Result
+
+----
+real    2m30.576s
+user    0m6.264s
+sys     0m0.372s
+----
+
+== Export from graph object
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword "CALL apoc.graph.fromDB('test',{})
+yield graph CALL apoc.export.cypher.graph(graph, 'yourPath/exportGraph.cypher',null)
+YIELD nodes, relationships, properties, file, source,format, time  return *" > 'yourPath/exportGraph'
+----
+
+=== Result
+
+.no config options
+----
+real    4m50.006s
+user    17m26.149s
+sys     0m13.145s
+----
+
+.cypher shell
+
+----
+real    5m6.467s
+user    19m14.328s
+sys     0m11.821s
+----
+
+.batch size 1.000
+
+----
+real    4m57.598s
+user    17m26.557s
+sys     0m13.465s
+----
+
+== Import Schema file
+
+[source,bash,subs=attributes]
+----
+time ./cypher-shell -u yourUsername -p yourPassword
+"call apoc.cypher.runSchemaFile('yourPath/exportPlain.schema.cypher')"
+----
+
+=== Result
+
+----
+real    0m0.683s
+user    0m0.896s
+sys     0m0.092s
+----
+
+== Import from file
```

Comment:
where are the import measurements from file via `cypher-shell < file.cypher` and `neo4j-shell -file file.cypher` ?
those are more interesting than runFile (which should also fix).